under-review/sdfobject-operational_state.sdf.json

```diff
@@ -0,0 +1,56 @@
+{
+  "info": {
+    "copyright": "copyright 2016-2017, 2019 Open Connectivity Foundation, Inc. All rights reserved.",
+    "license": "https://github.com/one-data-model/oneDM/blob/master/LICENSE",
+    "title": "Operational State",
+    "version": "2019-02-15"
+  },
+  "sdfObject": {
+    "operational.state": {
```

Comment:
Without going into details what is the right way to name this particular model, some general considerations:
- For the adopted models we should use form that is consistent to easily transform to ecosystem specific naming conventions. 
- Do we want to have structure here? Is "operational" super-type?
- ...

Reply:
this model originates from OCF, OCF has no issue with hierarchy.
how to remove the hierarchy (if needed).
note that it make sense to have some hierarchy in this model.

Reply:
I think the question was wether we are assigning any special meaning to a dot in a label.

Reply:
(More generally, we have the discussion how implementations should be mapping SDF names into their platforms/programming languages'/ecosystems' conventions.  E.g., DTDL doesn't like dashes, as it true for many programming languages.)

Reply:
the dot is there for readability

Reply:
> the dot is there for readability, since OCF did not want to use camecase in tag names

+1 to OCF here.  We have ".", "-" and "_" as typical separators.  We probably should have a convention which ones to use, and how to translate them to a snake_case world...